docker.html.md.erb

```diff
@@ -82,25 +82,29 @@ $ cf disable-feature-flag diego_docker
 
 Follow these instructions to deploy updated or new Docker images using the Cloud Foundry Command Line Interface (cf CLI).
 
-To perform the procedures in this topic, you must install cf CLI v6.13.0 or later. See <a href="../cf-cli/install-go-cli.html">Installing the cf CLI</a> for more information. 
+To perform the procedures in this topic, you must install cf CLI v6.13.0 or later. See <a href="../cf-cli/install-go-cli.html">Installing the cf CLI</a> for more information.
 For information about pushing a Docker image with a previous version of the cf CLI, see <a href= 'https://github.com/cloudfoundry-incubator/diego-design-notes/blob/master/docker-support.md'>Docker Support in CF + Diego</a>.
 
-When pushing an app, you specify either a Docker image on Docker Hub or a URI to a trusted Docker registry. Diego does not support using private registries. 
+When pushing an app, you specify either a Docker image on Docker Hub or a URI to a trusted Docker registry. Diego does not support using private registries.
 Also, when using Diego, you must use a registry that supports the <a href="https://docs.docker.com/registry/spec/api/">Docker Registry API V2</a>.
 
 ###<a id='public'></a>Push a Docker Image Using Docker Hub
 
 Run `cf push test-app -o cloudfoundry/MY-DOCKER-IMAGE` to deploy a Docker image. Replace `MY-DOCKER-IMAGE` with the name of an image from an accessible Docker Registry.
 
-For example, the following command pushes the `test-app` on Docker Hub to Cloud Foundry: 
+For example, the following command pushes the `test-app` on Docker Hub to Cloud Foundry:
 
 <pre class='terminal'>$ cf push test-app -o cloudfoundry/test-app</pre>
 
 ###<a id='private'></a>Push a Docker Image Using Docker Trusted Registries
 
 To deploy a Docker image using Docker trusted registries, run:
 
- <pre class='terminal'>$ cf push my-app -o MY-PRIVATE-REGISTRY.DOMAIN:5000/image/name:v2</pre> 
+ <pre class='terminal'>$ cf push my-app -o MY-PRIVATE-REGISTRY.DOMAIN:5000/image/name:v2</pre>
 
- Replace `MY-PRIVATE-REGISTRY.DOMAIN` with your domain name, which resolves to the private registry. 
+ Replace `MY-PRIVATE-REGISTRY.DOMAIN` with your domain name, which resolves to the private registry.
 In this example, 5000 is the port on which your private registry serves traffic. `image/name` is the name of the Docker image repository. `v2` is a tag in that repository that refers to a specific image.
+
+###<a id='volume'></a> Docker Volume Support
+
+Diego now supports Docker volumes. More information can be found [here](https://docs.cloudfoundry.org/devguide/services/using-vol-services.html) on how to enable volume support, and the limitations of NFS volumes within the [NFS Bosh Volume Release](https://github.com/cloudfoundry-incubator/nfs-volume-release#special-notes-for-docker-image-based-apps).
```

Comment:
@julian-hj @jeffpak The statement "Diego now supports Docker volumes" is confusing, and makes me think that CF does something magical when it sees a `VOLUME` directive in Docker image metadata when it may instead mean that the Diego cells are compatible with Docker volume plugins. It's now present at https://github.com/cloudfoundry/docs-dev-guide/blob/880131089e0c12a7e0b2b052c875f2c0ae37590e/deploy-apps/push-docker.html.md.erb#L144, but I think we should reword it to be clearer. /cc @seviet